src/app/core-module/actions/actions.component.html

```diff
@@ -1,2 +1,3 @@
-<app-search></app-search>
-<p>actions works!</p>
+<div id="add-course-block">
+  <button id="add-course-button" class="btn btn-sm btn-primary" (click)="addCourse()">+ Add course</button>
```

Comment:
You can use icons instead of "+". You don't need to fix it, it's just a little tip